app/ui-utils/client/lib/reactionListContent.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box, Tag, Divider } from '@rocket.chat/fuselage';
+
+import { useTranslation } from '../../../../client/contexts/TranslationContext';
+import Emoji from '../../../../client/components/basic/Emoji';
+
+export function Reactions(props) {
+	const styleDivider = {
+		marginTop: '8px',
+		marginBottom: '16px',
+	};
+
+	const reactionKeys = Object.keys(props.reactions);
+
+	return <Box>
+		{
+			reactionKeys.map((reaction, i) => <Box key={reaction}><div style={{ display: 'flex' }}><Emoji emojiHandle={reaction}/> {reaction}</div>
+				<Usernames usernames={props.reactions[reaction].usernames}></Usernames> { i !== (reactionKeys.length - 1) && <Divider key={reaction} style={styleDivider}/> } </Box>)
+		}
+	</Box>;
+}
+
+export function Usernames(props) {
+	const styleBox = {
+		paddingTop: '8px',
+	};
+
+	const styleTag = {
+		marginRight: '4px',
+	};
+
+	return <Box style={styleBox}>
```

Comment:
The `Box` component has some *styling props* that mimic CSS properties while compiling everything as a class name. You can use something like:

```suggestion
	return <Box paddingBlockStart='x8'>
```

This way increases performance by:

1. Not passing a new object at each render as prop, forcing the reconciliation between the React element and the target DOM element. Other primitive types like numbers and strings don't have this problem because equality is not referential for them, i.e., `{} === {}` is false while `'abc' === 'abc'` is true;
2. Creating a CSS rule at runtime and assigning it via `class` attribute, which is faster than serializing the style object into the string value of the `style` attribute.